src/hardware/dac.rs

```diff
@@ -146,14 +149,14 @@ impl Dac {
     /// * `shutdown` - TEC driver shutdown. True to set shutdown mode, false to reset shutdown mode.
     pub fn set_shutdown(&mut self, ch: Channel, shutdown: bool) {
         match (ch, shutdown) {
-            (Channel::Ch0, true) => self.gpio.shdn0.set_high().unwrap(),
-            (Channel::Ch1, true) => self.gpio.shdn1.set_high().unwrap(),
-            (Channel::Ch2, true) => self.gpio.shdn2.set_high().unwrap(),
-            (Channel::Ch3, true) => self.gpio.shdn3.set_high().unwrap(),
-            (Channel::Ch0, false) => self.gpio.shdn0.set_low().unwrap(),
-            (Channel::Ch1, false) => self.gpio.shdn1.set_low().unwrap(),
-            (Channel::Ch2, false) => self.gpio.shdn2.set_low().unwrap(),
-            (Channel::Ch3, false) => self.gpio.shdn3.set_low().unwrap(),
+            (Channel::Ch0, true) => self.gpio.shdn.0.set_high().unwrap(),
```

Comment:
@SingularitySurfer Isn't the polarity wrong here? To assert shutdown (`shutdown=true`) I want `SHDN` low.

Reply:
fixed/changed

Reply:
Ok, I was thinking in terms of the "enabled" state. `true` then means enabled. But we can do it the other way around.